pages/videos.tsx

```diff
@@ -0,0 +1,110 @@
+import PopupVideo from '@components/Video/PopupVideo'
+import VideoCard from '@components/Video/VideoCard'
+import { Fragment, useState } from 'react'
+import { Dialog, Transition } from '@headlessui/react'
+
+export interface Video {
+  youtubeVideoID: string
+  title: string
+}
+
+interface VideosGridProps {
+  videos: Array<Video>
+}
+
+const VideosGridView: React.FC<VideosGridProps> = ({ videos }) => {
```

Comment:
So sound like you want `({videos = DEFAULT_VIDEOS}) => {...}` instead https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters